Jenkinsfile-e2e

```diff
@@ -1,152 +1,69 @@
 
 // Run latest every 1 hour
 def cron_string = env.BRANCH_NAME == "latest" ? "0 */1 * * *" : ""
 
-def channels = [
-  simorgh: "#si_repo-simorgh",
-  testAlarms: "#simorgh-alarms-test",
-  liveAlarms:  "#simorgh-alarms-live"
-];
-
-def notifySlack(messageParameters, targetChannel) {
-  def message = "${messageParameters.emoji} Simorgh E2Es \n*Status:* ${messageParameters.buildStatus} \n*Env:* ${messageParameters.env} \n*Branch:* ${messageParameters.branch} \n*Pipeline:* ${env.BUILD_URL}"
-
-  slackSend(
-    channel: targetChannel,
-    color: messageParameters.colour,
-    message: message
-  )
-}
-
 def stageHasFailed = false;
 
 node {
-  properties(
-    [
-        buildDiscarder(
-            logRotator(
-                daysToKeepStr: '10',
-                artifactDaysToKeepStr: '10'
-            )
-        ),
-        pipelineTriggers([cron(cron_string)])
-    ]
-  )
-  timeout(time: 120, unit: 'MINUTES') {
-    withEnv([
-      'CI=true'
-    ]) {
-      cleanWs() // Clean the workspace
-      checkout scm // Checkout from git
-      docker.image("${nodeImage}").inside('--ipc host') {
-        try {
-          stage ('Install & Setup') {
-            sh 'make install'
-          }
-          stage ('Live E2Es') {
-            try {
-              sh 'make liveE2Es'
-            } catch (Throwable e) {
-              def messageParameters = [
-                buildStatus: 'Failed',
-                env: 'LIVE',
-                branch: env.BRANCH_NAME,
-                colour: 'danger',
-                emoji: '😱',
-              ]
-
-              if (env.BRANCH_NAME == 'latest') {
-                notifySlack(messageParameters, channels.liveAlarms)
-                stageHasFailed = true
-              }
-
-              currentBuild.result = 'FAILURE'
-              throw e
-            }
-          }
-          stage ('Local Prod Tests') {
-            try {
-              sh 'CYPRESS_SMOKE=false npm run build && npm run test:ci;'
-            } catch (Throwable e) {
-              def messageParameters = [
-                buildStatus: 'Failed',
-                env: 'LOCAL',
-                branch: env.BRANCH_NAME,
-                colour: 'danger',
-                emoji: '😱',
-              ]
-
-              if (env.BRANCH_NAME == 'latest') {
-                notifySlack(messageParameters, channels.liveAlarms)
-                stageHasFailed = true
-              }
-
-              currentBuild.result = 'FAILURE'
-              throw e
-            }
-          }
-        } catch (Throwable e) {
-          echo "The pipeline has failed with the error: ${e}"
-          currentBuild.result = 'FAILURE'
-          throw e
-        } finally {
-          def buildResult = currentBuild.result ?: 'SUCCESS'
-
-          echo "currentBuild.currentResult: ${currentBuild.currentResult}"
-          echo "currentBuild.result: ${currentBuild.result}"
+    properties(
```

Comment:
We're standardising on 4 spaces in Groovy everywhere now, I guess?

Reply:
ermmmmm clearly my editor preferences have kicked in here. I am happy with either 2 or 4?

Reply:
Absolutely no preference so long as it's consistent.